### DISCUSSION TITLE
Any news about on-demand (triggered via API route) revalidation instead of interval-based

### DISCUSSION THREAD

**Discussion by johnsonthedev** [vercel/next.js#17264] [Help] [answered]
[Here](https://github.com/vercel/next.js/discussions/11552#discussioncomment-2655), Joe said that they will be exploring on-demand (triggered via API route) revalidation instead of interval-based in the future.

I was wondering if they still plan to do this and if there is a post to subscribe to for updates ? Thx!

**Comment by lfades** [comment, marked as answer]
It's on our plans and at some point in the future we'll release a RFC for it. There's no defined date of release.

**Reply by kraegpoeth** [reply]
Is it fair to ask for an updated answer here half a year later - we would love this feature in our project! :)

**Reply by sinamoradi1375** [reply]
Same here ✌️

**Reply by ixartz** [reply]
Upvote this feature. Also needed ✌️

**Reply by timneutkens** [reply]
https://github.com/vercel/next.js/discussions/11552#discussioncomment-646963
